src/main/java/jimmy/mcgymmy/model/Model.java

```diff
@@ -21,12 +21,12 @@
     /**
      * Returns the current MacroList
      */
-    public MacroList getMacroList();
+    MacroList getMacroList();
```

Comment:
actually these should be public bc we access them from outside the `model`. idk why it still compiles though

Reply:
if im not wrong java interfaces methods are always assumed to be public. so we dont have to specify.

Reply:
https://docs.oracle.com/javase/tutorial/java/IandI/interfaceDef.html

^ Its stated in the docs ><